metadata-ingestion/src/datahub/utilities/sqlalchemy_query_combiner.py

```diff
@@ -333,7 +340,10 @@ def _execute_queue(self, main_greenlet: greenlet.greenlet) -> None:
             # Extract the results into a result for each query.
             index = 0
             for _, query_future in pending_queue.items():
-                cols = query_future.query.columns
+                query = query_future.query
+                if IS_SQLALCHEMY_1_4:
```

Comment:
Why is this part necessary? The subquery piece, whereas previously it was not

Reply:
not necessary - just suppresses a different warning that SA produces